src/player/player.rs

```diff
@@ -6,10 +6,12 @@ use librespot::core::config::SessionConfig;
 use librespot::core::keymaster;
 use librespot::core::session::{Session, SessionError};
 
+use librespot::playback::mixer::softmixer::SoftMixer;
+use librespot::playback::mixer::{AudioFilter, Mixer};
```

Comment:
AudioFilter is not used either apparently

Reply:
I'm pretty sure it's because of the trait but I'll fix it in the next commit